specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/mysql.json

```diff
@@ -0,0 +1,1662 @@
+          }
+        },
+        "properties":{
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          },
+          "readOnly": true,
+          "x-ms-client-flatten": false,
+          "description": "Additional descriptions for the operation."
+         }
+      },
+      "description": "REST API operation definition."
+    },
+    "OperationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "The list of resource provider operations."
+        }
+      },
+      "description": "A list of resource provider operations."
+    },
+    "LogFileProperties": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Log file name."
+        },
+        "sizeInKB": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Size of the log file."
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation timestamp of the log file."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last modified timestamp of the log file."
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of the log file."
+        },
+        "url": {
+          "type": "string",
+          "description": "The url to download the log file from."
+        }
+       },
+      "description": "The properties of a log file."
+    },
+    "LogFile": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the log file."
+        },
+        "properties": {
+          "$ref": "#/definitions/LogFileProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of the log file."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Represents a log file."
+    },
+    "LogFileListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LogFile"
+          },
+          "description": "The list of log files."
+        }
+      },
+      "description": "A list of log files."
+    },
+    "PerformanceTierServiceLevelObjectives": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ID for the service level objective."
+        },
+        "edition": {
+          "type": "string",
+          "description": "Edition of the performance tier."
+        },
+        "dtu": {
+          "type": "integer",
+          "description": "Database throughput unit associated with the service level objective"
+        },
+        "storageMB": {
```

Comment:
Still use dtu/storageMB in new biz model?

Reply:
missed in this one as well. Updated.